packages/@o3r/components/schematics/index.it.spec.ts

```diff
@@ -3,23 +3,23 @@
  * @jest-environment @o3r/test-helpers/jest-environment
  * @jest-environment-o3r-app-folder test-app-components
  */
-const o3rEnvironment = globalThis.o3rEnvironment;
```

Comment:
the point was to have this const linked to the "jsdoc" that we use to set up the jest environment